### ISSUE TITLE
Web-based serial console not working

### ISSUE THREAD

**Issue by stolendata** [scaleway/image-archlinux#37]
The web-based serial console on the server detail page doesn't work for Arch Linux on ARM/C1 servers. There is no echo on keyboard input, except rarely when you sometimes can enter text for 2-3 seconds before the console stops echoing. There is never any output coming back from the server; never a prompt or anything else that shows up, as if the keyboard input is never forwarded or received by Arch.

I have tried with Safari/OS X, Chromium/OS X and Firefox/Windows just to be sure it isn't a browser problem, but the result is the same.

**Comment by QuentinPerez** [comment]
Hi @stolendata,

I just tried and I haven't problems 😑 , can you retry and tell me if you have always this issue ?

**Comment by stolendata** [comment]
Yes, it's always. Everytime I open up the server page and the console the result is the same.

**Comment by QuentinPerez** [comment]
Did you start a new instance ?

**Comment by stolendata** [comment]
I can confirm that the console works for the new Arch Linux build for C1. Can you please elaborate on what changes you have made, so that I can incorporate them in my old install?

**Comment by QuentinPerez** [comment]
I changed the kernel to use `4.5.7 std`

**Comment by stolendata** [comment]
@QuentinPerez Forgive me for asking again, but surely there must be something else changed? I am running 4.5.7 now (since 4.4.8 stopped booting after some update), on a fully updated Arch made from the old build I installed about 4 months ago, and on my installation the serial console *does not* work, while on an install from your new Arch build it will work.
